### PR TITLE
Modify About Us Page

### PR DESCRIPTION
**Pull request by iamrohitsuthar**

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -57,284 +63,292 @@
...
-                     <p class="services-right-text">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries,</p>
-                  </div>
+               <!-- <div class="">
+                  <a href="#nogo" class="hero__play"> <img src="assets/images/img-2.png" class="video-a"></a>
+               </div> -->
+            </div>
+            <div class="col-md-6 wow fadeInUp">
+               <div class="services-right">
+                  <p class="about_bottom_r">Mission & Vision </p>
+                  <p class="services-right-text">Our mission is to help enterprises accelerate adoption of new technologies, untangle complex issues that always emerge during digital evolution, and orchestrate ongoing innovation. Whether it is a consumer-oriented app or a transformative enterprise-class solution, the company leads the process from ideation and concept to delivery, and provides ongoing support.</p>
```
Comment (saglaniharsh) [inline about.html]: Our mission is to help enterprises accelerate adoption of new technologies, untangle complex issues that always emerge during digital evolution, and orchestrate ongoing innovation. Whether it is a consumer-oriented app or a transformative enterprise-class solution, we believe to lead the process from ideation and concept to delivery, and provide ongoing support.

File: about.html [hunk 2]
```diff
@@ -57,284 +63,292 @@
...
-                       
-                     </div>
-                     <div class="col-md-5">
-                        <div class="top-banner wow fadeInRight text-left" style="visibility: visible; animation-name: fadeInRight;">
-                           <a id="#services" href="contact.html" class="btn btn-primary  wow fadeInUp  js-scroll-trigger" data-wow-delay="1s" style="visibility: visible; animation-delay: 1s; animation-name: fadeInUp;">CONTACT SALES</a>
-                        </div>
+                  <div class="col-md-5">
+                     <div class="top-banner wow fadeInRight text-left"
+                        style="visibility: visible; animation-name: fadeInRight;">
+                        <a id="#services" href="contact.html" class="btn btn-primary  wow fadeInUp  js-scroll-trigger"
```
Comment (saglaniharsh) [inline about.html:255]: change button style

**Review by saglaniharsh — CHANGES_REQUESTED** [review]
A little change in the text of mission & vision and change the button above the footer;
And please use other image from unsplash.com

**Comment by iamrohitsuthar** [comment]
> A little change in the text of mission & vision and change the button above the footer;
> And please use other image from unsplash.com

Hey, It would be great if you please share the URL of the images. I had previously searched but unable to find any relevant images.

**Comment by saglaniharsh** [comment]
> 
> 
> > A little change in the text of mission & vision and change the button above the footer;
> > And please use other image from unsplash.com
> 
> Hey, It would be great if you please share the URL of the images. I had previously searched but unable to find any relevant images.

[Select appropriate images related to tech from these](https://unsplash.com/s/photos/corporate)

**Comment by iamrohitsuthar** [comment]
Please check now!

**Review by saglaniharsh — APPROVED** [review]
Just change the envelope-o to envelope icon and done.

**Pull request merged**